src/projections/getProjectionStatus.ts

```diff
@@ -0,0 +1,39 @@
+import { Client } from "../Client";
+import type { ProjectionStatus } from "../types";
+import { debug } from "../utils";
+import type { GetProjectionStatisticsOptions } from "./getProjectionStatistics";
+
+export interface GetProjectionStatusOptions
+  extends GetProjectionStatisticsOptions {}
+
+declare module "../Client" {
+  interface Client {
+    /**
+     * Gets the projection's current status.
+     * @param projectionName The name of the projection.
+     * @param options Get status options.
+     */
+    getProjectionStatus(
+      projectionName: string,
+      options?: GetProjectionStatusOptions
+    ): Promise<ProjectionStatus>;
+  }
+}
+
+Client.prototype.getProjectionStatus = async function (
+  this: Client,
+  projectionName: string,
+  baseOptions: GetProjectionStatusOptions = {}
+): Promise<ProjectionStatus> {
+  debug.command("getProjectionStatus: %O", {
+    projectionName,
+    options: baseOptions,
+  });
+
+  const { projectionStatus } = await this.getProjectionStatistics(
```

Comment:
**SUGGESTION:** Shouldn't we mark `getProjectionStatistics` and `GetProjectionStatisticsOptions` as obsolete/deprecated?

Reply:
They're still a supported operation: https://github.com/EventStore/architecture-and-planning/blob/main/rfcs/021%20-%20gRPC%20Projections.md#supported-operations

Reply:
Hm 🤔 that's weird.